swiftnav/amb_kf.pyx

```diff
@@ -499,62 +262,3 @@ def get_kf_from_alms_using_sdiffs(phase_var, code_var,
   memcpy(&(pykf.kf), &kf, sizeof(amb_kf_c.nkf_t))
 
   return pykf
-  
-# def get_kf_from_alms(phase_var, code_var,
-#                      pos_trans_var, vel_trans_var, int_trans_var,
-#                      alms, GpsTime timestamp, ref_ecef, dt):
-#   n = len(alms)
-#   state_dim = n + 5
-#   obs_dim = 2 * (n-1)
-
-#   cdef almanac_t al[32]
-#   cdef almanac_t a_
-#   cdef np.ndarray[np.uint8_t, ndim=1, mode="c"] prns = \
-#         np.empty(n, dtype=np.uint8)
-#   for i, a in enumerate(alms):
-#     a_ = (<Almanac> a).almanac
-#     prn_ = (<Almanac> a).prn
-#     memcpy(&al[i], &a_, sizeof(almanac_t))
-#     prns[i] = (<Almanac> a).prn
-
-#   cdef np.ndarray[np.double_t, ndim=1, mode="c"] ref_ecef_ = \
-#     np.array(ref_ecef, dtype=np.double)
-
-#   cdef gps_time_t timestamp_ = timestamp.gps_time
-
-#   cdef amb_kf_c.nkf_t kf = amb_kf_c.get_kf_from_alms(phase_var, code_var,
-#                                                         pos_trans_var, vel_trans_var, int_trans_var,
-#                                                         n, &al[0], timestamp_, &ref_ecef_[0], dt)
-
-#   cdef KalmanFilter pykf = KalmanFilter()
-#   memcpy(&(pykf.kf), &kf, sizeof(amb_kf_c.nkf_t))
-
-#   return pykf
-
-  # cdef np.ndarray[np.double_t, ndim=2, mode="c"] transition_mtx = \
-  #       np.empty((state_dim, state_dim), dtype=np.double)
-
-  # cdef np.ndarray[np.double_t, ndim=2, mode="c"] transition_cov = \
-  #       np.empty((state_dim, state_dim), dtype=np.double)
-
-  # cdef np.ndarray[np.double_t, ndim=2, mode="c"] decor_mtx = \
-  #       np.empty((n-1, n-1), dtype=np.double)
-
-  # cdef np.ndarray[np.double_t, ndim=2, mode="c"] decor_obs_mtx = \
-  #       np.empty((obs_dim, state_dim), dtype=np.double)
-
-  # cdef np.ndarray[np.double_t, ndim=1, mode="c"] decor_obs_cov = \
-  #       np.empty(obs_dim, dtype=np.double)
-
-  # memcpy(&transition_mtx[0,0], kf.transition_mtx, state_dim * state_dim * sizeof(double))
-  # memcpy(&transition_cov[0,0], kf.transition_cov, state_dim * state_dim * sizeof(double))
-  # memcpy(&decor_mtx[0,0], kf.decor_mtx, (n-1) * (n-1) * sizeof(double))
-  # memcpy(&decor_obs_mtx[0,0], kf.decor_obs_mtx, obs_dim * state_dim * sizeof(double))
-  # memcpy(&decor_obs_cov[0], kf.decor_obs_cov, obs_dim * sizeof(double))
-
-  # return KalmanFilter(prns,
-  #                     transition_mtx,
-  #                     transition_cov,
-  #                     decor_mtx,
-  #                     decor_obs_mtx,
-  #                     decor_obs_cov)
```

Comment:
Same question on this file - looks like this has mostly been deleted. Is that okay? Where does this functionality go, or is it no longer necessary?